sulis_view/src/item_button.rs

```diff
@@ -258,15 +258,21 @@ impl ItemButton {
         // window may change after the owing inventory window is opened
         let (root, root_view) = Widget::parent_mut::<RootView>(widget);
         if let Some(window_widget) = root_view.get_merchant_window(&root) {
-            let merchant_window = Widget::kind_mut::<MerchantWindow>(&window_widget);
-
-            let action = ButtonAction {
-                label: "Sell".to_string(),
-                callback: sell_item_cb(merchant_window.player(), item_index),
-                can_left_click: true,
-            };
-
-            Some(action)
+            let merchant = Widget::kind_mut::<MerchantWindow>(&window_widget).player();
+
+            if self.get_item_state().filter(|i| !i.identified).is_some() {
```

Comment:
I am ok with changing the sell button to identified for a first pass.  But there is no feedback to the user at all about how much the identification will cost.

Reply:
I see, to be storing "unidentified" several times in adjectives, which causes problems.

Reply:
this is fixed